examples/exchange-files-in-browser/public/index.html

```diff
@@ -62,7 +62,7 @@
     </div>
 
     <!-- The IPFS node module -->
-    <script src="//unpkg.com/ipfs/dist/index.min.js"></script>
+    <!-- <script src="//unpkg.com/ipfs/dist/index.min.js"></script> -->
```

Comment:
This was supposed to be the example that showed that "no extra batteries required". Kind of sad we lost that.

Reply:
@diasdavid unpkg doesn't allow linking to a branch? We could also publish latest master ourselves on IPFS and link that instead

Reply:
We don't have to keep this, with the new release that would be fine to use the unkpkg version again